cache.js

```diff
@@ -9,9 +9,16 @@ const path = require('path')
  * @param {number} repoID repository identifier
  * @param {number} prID pull request identifier
  * @param {string} branchTag branch name
+ * @param {string} app Optional: an app that calls the function. (default bandit)
+ * It's needed because gosec uses GOPATH.
  */
-function getBranchPath (repoID, prID, branchTag) {
-  return path.join('cache', repoID.toString(), prID.toString(), branchTag)
+function getBranchPath (repoID, prID, branchTag, app) {
+  app = app || 'bandit'
```

Comment:
Wouldn't it be easier to set 'cache' as the GOPATH, then create a 'src' subfolder inside? That way we don't have to deal with different access paths for each linter